docs/release-notes/14-4-0/README.md

```diff
@@ -0,0 +1,224 @@
+
+For upcoming meetings, the tab will display them in chronological order with the nearest occurrence first and the most distant last. For past meetings, the tab will display them in reverse chronological order, with the most recent occurrence first and the oldest last.
+
+### Design: Improve avatar color generation so that users with same names are distinguishable
+
+Some users share the same name or create several accounts with the same name. To be able to tell them apart, e.g. when looking through search results for that name, we improved the avatar color generation, so that **users with the same name are displayed with different colors**. We recommend uploading an avatar image so that the distinction is even more obvious.
+
+### Dropdown menu on work packages: Move "Copy link to clipboard" up
+
+To avoid confusion with the "copy" or "copy to another project" actions, we have moved the "copy link to clipboard" action upwards in the drop-down menu for work packages. This action is now in third place in the work package table view. In the work package detail view, the action is now in first position.
+
+![Detailed view of a work package, drop down menu with highlighted 'Copy link to clipboard'](openproject-14-4-copy-link.png)
+
+### Add link to storage provider in storage edit view
+
+To improve efficiency and access to file storages, OpenProject 14.4 allows administrators to click on a **direct link to the storage provider in the edit view of the storage**.
+
+In the "General information" section, there is now a link that opens in a new window and leads to the root folder of the storage. For Nextcloud, this link refers to the root folder of the Nextcloud instance. For OneDrive/SharePoint, the link refers to the root folder of the drive ID and also works with "OneDrive Business".
+
+See https://community.openproject.org/wp/56045 for more information on this feature.
+
+### Show changes of long text custom fields in the activity similar to changes in the description
+
+To keep the activity view cleaner when multiple changes are made to long text custom fields, OpenProject 14.4 now **displays updates to long custom fields in a similar format to description updates**. As a user watching the activity, you will see an indication of changes to long text custom fields, making it easier to manage and review activities.
+
+In the activity tab, updates to long text custom fields will be displayed as in the following example – 'Remarks' being a custom field:
+
+![Example screenshot showing how changes on a custom field will now look like in the work package activity tab](Custom_field__long_text__changed.png)
+
+Clicking on the "Details" link will take the user to the diff view, which is also used for work package descriptions and project status explanations.
+
+<!--more-->
```

Comment:
You could add the link to the docs: /user-guide/projects/project-settings/custom-fields/ and/or
/docs/system-admin-guide/custom-fields/